src/javascript/binary/websocket_pages/user/new_account/account_type.js

```diff
@@ -0,0 +1,49 @@
+const Client           = require('../../../base/client');
+const BinaryPjax       = require('../../../base/binary_pjax');
+const urlFor           = require('../../../base/url').urlFor;
+const BinarySocket     = require('../../socket');
+
+const AccountType = (() => {
+    let url_real,
+        url_ico,
+        container;
+    const onLoad = () => {
+        BinarySocket.wait('landing_company').then((response_lc) => {
+            url_ico  = `${urlFor('new_account/realws')  }#ico`;
+            url_real = urlFor(Client.getUpgradeInfo(response_lc).upgrade_link);
+            container = document.getElementById('account_type_container');
+
+            if(Client.canOpenICO()) {
+                container.setVisibility(1);
+                addEventListener();
+            } else {
+                BinaryPjax.load(url_real);
+            }
+        });
+
+        const addEventListener = () => {
+            $(container)
+                .find('#btn_submit')
+                .off('click')
+                .on('click', () => {
+                    $(container)
+                        .find('input[type=radio]')
```

Comment:
can we store this outside the event listener? e.g. on line 25 `const $radio = $(container).find('input[type=radio]');`